app/lib/widgets/alpha_version_banner.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+/// Displays a [Banner] saying "ALPHA" when running an alpha version at top left
+/// hand corner.
+///
+/// Does nothing when running a beta or stable version.
+///
+/// To be able to detect if this app is running an alpha version, set is it
+/// required to set the environment variable "ALPHA" set to true (like `flutter
+/// run --dart-define ALPHA=true`).
+///
+/// This widget is similar and inspired by the [CheckedModeBanner] which display
+/// "DEBUG" at the top right hand corner when running a Flutter app in debug
+/// mode.
+class AlphaVersionBanner extends StatelessWidget {
+  /// Creates a const alpha version banner.
+  const AlphaVersionBanner({
+    Key key,
+    @required this.child,
+    this.isAlphaVersion = const bool.fromEnvironment('ALPHA'),
```

Comment:
What do you think about using `IS_ALPHA` instead of `ALPHA` for the environment variable?   

Or what about `--dart-define DEVELOPMENT_STAGE=ALPHA`?

Reply:
`ALPHA` is also okay for me, would just like to hear your opinion

Reply:
`DEVELOPMENT_STAGE=ALPHA` might have more option for future, like `BETA`, etc and has not the problem that `IS_BETA` and `IS_ALPHA` could be true.